lib/core/src/Cardano/Wallet/DB/Sqlite/CheckpointsOld.hs

```diff
@@ -718,68 +704,3 @@ selectRndStatePending wid = do
   where
     assocFromEntity (RndStatePendingAddress _ accIx addrIx addr) =
         ((W.Index accIx, W.Index addrIx), addr)
-
```

Comment:
Reviewer's note, ouroborus-network provides it's own ReaderT instance for MonadSTM (https://github.com/input-output-hk/ouroboros-network/blob/ad47441e9e399495579d7dd05ccaf96b46a487b2/io-classes/src/Control/Monad/Class/MonadSTM.hs#L897)